kolibri/core/assets/src/riot-tags/usermgmt-user.tag.html

```diff
@@ -5,8 +5,9 @@
   </div>
   <div class="button-group">
     <button onclick={delete}>delete</button>
-    <button disabled>edit</button>
+    <button onclick={showEditForm}>edit</button>
   </div>
+  <usermgmt-editform user={getUser(username)}></usermgmt-editform>
```

Comment:
This is an example where I don't like riot's built-in looping semantics. I would prefer it if riot attached the `user` object to the `opts` object rather than magically dropping `user`'s attributes onto the `tag` object.